browser_patches/firefox/juggler/content/NetworkMonitor.js

```diff
@@ -33,16 +33,18 @@ class NetworkMonitor {
       const frame = this._frameTree.frameForDocShell(window.docShell);
       if (!frame)
         return;
-      this._requestDetails.set(httpChannel.channelId, {
+      const typeId = httpChannel.loadInfo ? httpChannel.loadInfo.internalContentPolicyType : 1;
```

Comment:
... : Ci.nsIContentPolicy.TYPE_OTHER; to match the other place ?

Reply:
Done.